### DISCUSSION TITLE
Updates more freequent than once per 4 minutes

### DISCUSSION THREAD

**Discussion by harentius** [NightscoutFoundation/xDrip#2543] [General]
Hello
May be quite naive question...
Why does xdrip have a fixed sync period of 4 (or 5?) minutes?
Freestyle Libre 3 sends data every minute.
xdrip has bigger latency, and also smoothing data over Libre 3 smoothing, which looks like even bigger latency.

There is an ability to show “Raw Libre 2 values” which partially solves the issue, but:

1. You need to open plot to see this
2. Data is not transferred to AAPS

So can there be some “raw mode”, which would make xdrip don’t smooth data, and display/forecast it more frequently?

P.S. The discussion was [here](https://github.com/NightscoutFoundation/xDrip/issues/1474#issuecomment-1324285569) too.

**Comment by PhilTU** [comment]
The 5 m,inutes was built in from the beginning, including in Nightscout, as I understand it. This was thought to be adequate and at the time was the highest frequency data from available sensors. The Libre of course has 1 minute values from the beginning but the transmitter devices only send every 5, then with Libre 2 the patched app also only provides a value every 5 minutes. 

xDrip+ receives data every minute from FS Libre 2 via Bluetooth using OOP2 to decode.

**Reply by harentius** [reply]
Is with OOP2 data updated on plot and broadcasted to AAPS every minute?

So you are saying tha because of initial architecture it is very hard to reduce or configure this period?

**Comment by jamorham** [comment]
It is possible for data to be processed more frequently than every 5 minutes (eg every minute) but this is not yet fully implemented and some external data services like AAPS expect data every 5 minutes as far as I know so we have to maintain compatibility there. There are some other things that break with one minute readings. It is currently a work in progress.

**Reply by harentius** [reply]
Thank you so much for the clarification! 👍

**Comment by Mdullaway** [comment]
Is there any update on this please?

I'm using libre 3 with camaps (so not through the libre app) so need to use the "hardware data source - companion app" in xdrip but would like to display the one minute values that are being generated.

Many thanks.
